README.md

```diff
@@ -141,7 +140,7 @@ Yes, but we already have to agree on base encodings, so this is not hard. The ta
 
 ## Disclaimers
 
-Warning: **obviously multibase changes the first byte(s) depending on the encoding**. Do not expect the value to be exactly the same. Remove the multibase prefix before using the value.
+Warning: **obviously multibase changes the first symbol depending on the encoding**. Do not expect the value to be exactly the same. Remove the multibase prefix before using the value.
```

Comment:
This means that the `<multibase-encoding-symbol>` will always only be a single character. Isn't this opposed to the future-proofness we have in other multiformats?

If it really is a single character only, could we rename it from `symbol` to `character`? I think it would make things clearer.

Reply:
I say "symbol" as I really don't think of, e.g., emoji, as characters. However, I guess they technically are.

We could also generalize this to "one or more characters". However, I'm really not sure if we want to do that. In *that* case, I'd:

1. Define a multibase for the first character.
2. Define a base encoding that interprets the rest.

Alternatively, we could just say "currently" and give us room to upgrade later. Really, as long as we're unambiguous, we can always extend this.

Reply:
Having a single character only makes things certainly way easier and the need for having multiple seems like a pretty artificial problem. So the upgrade path, if we ever need it would then be defining one character that means "hey there's more characters" and we are good.